src/main/java/calculator/StringCalculator.java

```diff
@@ -11,43 +10,29 @@
  * @since 2020-02-07
  */
 public class StringCalculator {
+
     private static final Pattern CUSTOM_PATTERN = Pattern.compile("//(.)\n(.*)");
+    public static final String DEFAULT_DELIMITER = ",|;";
```

Comment:
`StringCalculator` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private static final String DEFAULT_DELIMITER = ",|;";
```

Reply:
앗 이것을 놓쳤네요 ㅠ